jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -64,21 +64,21 @@ class CubevizProfileView(SpecvizProfileView):
              'jdaviz:boxzoom', 'jdaviz:xrangezoom',
              'bqplot:panzoom', 'bqplot:panzoom_x',
              'bqplot:panzoom_y', 'bqplot:xrange',
-             'jdaviz:selectslice']
+             'jdaviz:selectslice', 'jdaviz:selectline']
 
     # categories: zoom resets, zoom, pan, subset, select tools, shortcuts
     tools_nested = [
                     ['bqplot:home'],
                     ['jdaviz:xrangezoom', 'jdaviz:boxzoom'],
                     ['bqplot:panzoom', 'bqplot:panzoom_x', 'bqplot:panzoom_y'],
                     ['bqplot:xrange'],
-                    ['jdaviz:selectslice'],
+                    ['jdaviz:selectslice', 'jdaviz:selectline'],
                     ['jdaviz:sidebar_plot', 'jdaviz:sidebar_export']
                 ]
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self._initialize_toolbar_nested(default_tool_priority=['jdaviz:selectslice'])
+        self._initialize_toolbar_nested(default_tool_priority=self.tools_nested[4])
```

Comment:
this makes it so clicking home will now default to either the select slice or select line tool (based on which one is the current "primary" entry in the toolbar)

Reply:
Will this magic number 4 every change? Maybe a comment to explain what "4" is for future reference?